code/modules/reagents/chemistry/machinery/chem_master.dm

```diff
@@ -1,3 +1,6 @@
+#define AUTO_STYLE "auto"
+#define FALLBACK_STYLE "_"
```

Comment:
You should define this outside so that you can use it in asset_list_items.dm as well.

Reply:
They feel rather local tho. Where global constants of this kind should be defined?

Reply:
Where you put it is fine.